frontend/src/main/java/org/abs_models/frontend/delta/Copy.jadd

```diff
@@ -0,0 +1,35 @@
+aspect Copy {
```

Comment:
This aspect has a very generic filename and name and seems to have a very specific use.  

Can you add a comment saying what this one does differently / in addition to ASTNode.copy(), ASTNode.treeCopy[NoTransform](), and when and how to use it, and maybe rename it?